includes/base/widget-base.php

```diff
@@ -72,6 +165,14 @@ public function start_controls_section( $section_id, array $args ) {
 		}
 	}
 
+	/**
+	 * Register widget skin control.
+	 *
+	 * Used to add a new skin controls to the widget. Added at the top of the
```

Comment:
An internal method that is added a skin control to the widget. Added at the top of the...